AcmeClient.php

```diff
@@ -222,6 +222,19 @@ public function requestCertificate($domain, CertificateRequest $csr, $timeout =
         return $this->finalizeOrder($order, $csr, $timeout);
     }
 
+    public function requestAlternateCertificate($domain, CertificateRequest $csr, $timeout = 180)
+    {
+        Assert::stringNotEmpty($domain, 'requestCertificate::$domain expected a non-empty string. Got: %s');
+        Assert::integer($timeout, 'requestCertificate::$timeout expected an integer. Got: %s');
+
+        $order = $this->requestOrder(array_unique(array_merge([$domain], $csr->getDistinguishedName()->getSubjectAlternativeNames())));
+
+        /** @var \AcmePhp\Core\Protocol\CertificateOrder $order */
+        $order = $order->returnAlternateCertificateIfAvailable();
+
+        return $this->finalizeOrder($order, $csr, $timeout, true);
+    }
+
```

Comment:
not using this method in our client, but adding it for other clients that want to request an alternate certificate but are not calling `finalizeOrder` directly with a `CertificateOrder`